.github/workflows/plugins-ci.yml

```diff
@@ -89,6 +94,26 @@ jobs:
       - name: Run tests
         run: npm test
 
+      - name: Coveralls Parallel
+        if: inputs.coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
```

Comment:
This is allowed. See: https://docs.github.com/cn/actions/using-workflows/reusing-workflows#overview
<img width="738" alt="image" src="https://user-images.githubusercontent.com/8198408/176582019-191ab2f2-c3ac-471a-a657-c488a4ac0ce4.png">